### PR TITLE
Sohaib status model, Alembic Scripts

### PR DESCRIPTION
**Pull request by SohaibkhanFolio3**
Alembic Setup with respect of status is done , 
New Status_id column  replaces status column in Copy Table , 
status_id coming from status table now , table created . 
Tickets:
https://app.asana.com/0/1204411152293714/1204561574917319/f
https://app.asana.com/0/1204411152293714/1204468697159503/f
https://app.asana.com/0/1204411152293714/1204610448737250/f
https://app.asana.com/0/1204411152293714/1204593647129122/f

### CODE REVIEW

**Review by mustansirMuz — APPROVED** [review]
LGTM

**Review by Sana-Ullah786 — CHANGES_REQUESTED** [review]
No PUT method for status ?

File: src/endpoints/status/delete/status_delete.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+from sqlalchemy import delete
+from sqlalchemy.orm import Session
+from starlette import status
+
+from src.dependencies import get_db
+from src.endpoints.status.router_init import router
+from src.models.status import Status
+
+
+@router.get("/", status_code=status.HTTP_204_NO_CONTENT, response_model=None)
```
[the remark this answers is not in the document]
Comment (Sana-Ullah786) [reply src/endpoints/status/delete/status_delete.py:13]: Fixed.

File: src/models/all_models.py [hunk 2]
```diff
@@ -7,3 +7,14 @@
 from src.models.genre import *  # noqa
 from src.models.language import *  # noqa
 from src.models.user import *  # noqa
+
```
[the remark this answers is not in the document]
Comment (Sana-Ullah786) [reply src/models/all_models.py:10]: Fixed.

**Review by Sana-Ullah786 — APPROVED** [review]
Approved. ✨

**Pull request merged**